### PR TITLE
Just some more error checking.

### PR DESCRIPTION
**Pull request by rudigens**

### CODE REVIEW

File: src/write_hdf5_xml/write_hdf5_xml.c [hunk 1]
```diff
@@ -2148,8 +2163,9 @@ int main(int argc, char **argv)
       " sensor\">%s</beam_mode>\n", meta->general->mode);
     fprintf(fp, "      <absolute_orbit type=\"int\" definition=\"absolute orbit"
       " of the image\">%d</absolute_orbit>\n", meta->general->orbit);
-    fprintf(fp, "      <frame type=\"int\" definition=\"frame number of the "
-      "image\">%d</frame>\n",  meta->general->frame);
+    if (meta->general->frame > 0)
```
Comment (khogenso) [inline src/write_hdf5_xml/write_hdf5_xml.c:2168]: Do we need >= 0 here?  Is 0 a valid frame number?

File: src/write_hdf5_xml/write_hdf5_xml.c [hunk 2]
```diff
@@ -2659,6 +2706,10 @@ int main(int argc, char **argv)
       sprintf(data_source, "ALOS PALSAR");
       sprintf(copyright, "JAXA, METI (%d)", year);
     }
+    else if (strncmp_case(meta->general->sensor, "ERS", 3) == 0) {
+      sprintf(data_source, "%s SAR", meta->general->sensor);
+      sprintf(copyright, "ESA (%d)", year);
+    }
```
Comment (khogenso) [inline src/write_hdf5_xml/write_hdf5_xml.c:2714]: else
  asfPrintError("Cannot set data source or copyright for %s!", meta->general->sensor)

File: src/write_hdf5_xml/write_hdf5_xml.c [hunk 3]
```diff
@@ -2118,17 +2124,26 @@ int main(int argc, char **argv)
...
-    if (params->input_VH_file && params->input_VV_file) {
-      strcpy(beam_mode, "PLR");
-      sprintf(original_file, "%s,%s,%s,%s", params->input_HH_file, 
-        params->input_HV_file, params->input_VH_file, params->input_VV_file);
+    else {
+      strcpy(beam_mode, "STD");
+      if (params->input_HH_file)
+        strcpy(original_file, params->input_HH_file);
+      else if (params->input_VV_file)
+        strcpy(original_file, params->input_VV_file);
```
Comment (khogenso) [inline src/write_hdf5_xml/write_hdf5_xml.c:2146]: else
 asfPrintError()
needed here too

File: src/write_hdf5_xml/write_hdf5_xml.c (hunk 1 above)
Comment (rudigens) [inline src/write_hdf5_xml/write_hdf5_xml.c:2168]: As far as I know 0 is not a valid frame number.

Comment (khogenso) [inline src/write_hdf5_xml/write_hdf5_xml.c:2168]: Ok, good, so we are ok there.

**Pull request merged**